### ISSUE TITLE
How would SD-JWT be used in claims transcription

### ISSUE THREAD

**Issue by PieterKas** [oauth-wg/oauth-identity-chaining#108]
The claims transcription section suggest that SD-JWT can be used for selective disclosure, but does not provide additional information nohow to combine this draft with something like SD-JWT. Do we need to provide more information on how this could be done (e.g. would the SD-JWT be requested using the assertion grant)?

**Comment by PieterKas** [comment]
@bc-pi @kburgin3 @arndt-s although I agree that SD-JWT could be used, it is not clear to me how that would be achieved in the context of the identity chaining draft. I am worried that this may be quite a bit of scope expansion if we try to explain how it gets done, and not altogether certain we need it as a part of a "basic framework". Perhaps if someone feels they need to support SD-JWTs with identity chaining, they should create aa profile or extension.

**Comment by kburgin3** [comment]
I agree we do not need to include text around SD-JWT and rather leave it to a profile or extension.

**Comment by PieterKas** [comment]
After discussion amongst editors, it was decide to remove this reference as the underlying specifications for assertion grant is scoped to JWT and using SD-JWTs for access tokens is undefined, and also incompatible with the idea of opaque tokens since clients are not meant to be inspecting or selectively disclosing them.
